examples/with-backup/README.md

```diff
@@ -0,0 +1,82 @@
+growi-docker-compose with backup example
+========================================
+
+# Introduction
+
+Growi has MongoDB to store own pages data and users data.
+
+This guide show how to backup MongoDB in case of using [weseek/growi-docker-compose](https://github.com/weseek/growi-docker-compose).
+
+# How to backup with "weseek/mongodb-awesome-backup"
+
+We show how to backup in case of you use Growi in docker-compose environment.
+
+## Requirements
+
+- AWS S3 buckets and IAM account to store backup files
+
+## Quick start
+
+1. Override `docker-compose.override.yml` to your `docker-compose.yml`
+2. Create file `.env/SOME_FILE_NAME` or copy `.env/s3-secret-keys`, and set environment about secret ID and access key of S3.
```

Comment:
Create file `.env/SOME_FILE_NAME` or copy `.env/s3-secret-keys` to set environment variables for AWS.